Signal/src/call/SignalCall.swift

```diff
@@ -41,6 +41,16 @@ protocol CallDelegate: class {
     var state: CallState {
         didSet {
             Logger.debug("\(TAG) state changed: \(oldValue) -> \(state)")
+
+            // Update connectedDate
+            if state == .connected {
+                if connectedDate == nil {
+                    connectedDate = NSDate()
+                }
+            } else {
```

Comment:
Will this immediately wipe the call duration when the call ends?

ux nit: The call view hangs out for a second when the call was hung up by the remote party. I think we should continue to display the time (frozen) after the other party hangs up.